docs/developer-docs/latest/setup-deployment-guides/configurations/required/admin-panel.md

```diff
@@ -42,7 +42,7 @@ The `./config/admin.js` file should at least include a minimal configuration wit
 :::
 
 :::: tabs card
-::: tab Minimal configuration
+::: tab Minimal configuration JavaScript
```

Comment:
I think it could be cleaner/prettier to use VuePress code groups for this.

So we would have:
- minimal configuration tab
  - code block for JS
  - code block for TS
- full configuration tab
  - code block for JS
  - code block for TS